pyproject.toml

```diff
@@ -30,6 +30,7 @@ pytest-mock = "^3.7.0"
 
 [tool.pytest.ini_options]
 minversion = "6.0"
+addopts = "--ignore=test/integration_tests/flavors --ignore=test/integration_tests/test_container"
```

Comment:
Please add a rationale why the tests are ignored

Reply:
These are resources and no tests, I will put these two directories into a resources directory